doc/makeqbk.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/sh
```

Comment:
This script should have execute attributes. Cherry pick this to change: https://github.com/seelabs/Beast/commit/a4d840607f7bcadd52f183c69891b7c50cdd8c1e